code-challenges-401/arrayShift/array-shift.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const array = [1,2,7,9];
+
+const insertShiftArray = (arr, value) => {
+  let newArray = [];
+  const middleIndex = Math.ceil(arr.length/2);
+  for(let i=0;i<arr.length;i++){
+    if(i === middleIndex){
+          newArray.push(value);
+          newArray.push(arr[i]);
+    }else{
```

Comment:
This looks really good, there is a way to do this without the .push() which would good knowledge to have!